src/proxy/http/remap/RemapConfig.cc

```diff
@@ -100,6 +97,14 @@ BUILD_TABLE_INFO::BUILD_TABLE_INFO()
 BUILD_TABLE_INFO::~BUILD_TABLE_INFO()
 {
   this->reset();
+
+  // clean up any leftover named filter rules
+  auto *rp = rules_list;
+  while (rp != nullptr) {
+    auto *tmp = rp->next;
+    delete rp;
+    rp = tmp;
+  }
```

Comment:
Awesome.

Let's not do this in this PR because what you're doing here with delete appropriately fits the current setup, but it would be good to change these over to shared_ptrs. (I think they are shared across rules so unique_ptr won't work?).

Reply:
They aren't actually shared, but copied into the actual mappings as they are created.  This code is very pointer-heavy and is over due for a general update to C++ pass.  I do plan on expanding this work soon, but wanted to get these couple fixes that will need to be back ported.  Thanks for the review!